lightly/api/api_workflow_client.py

```diff
@@ -97,7 +97,7 @@ def __init__(
 
         configuration = get_api_client_configuration(token=token)
         self.api_client = ApiClient(configuration=configuration)
-        self.user_agent = f"Lightly/{__version__}/python ({platform.platform()})"
```

Comment:
how does this allow one to pass the worker version together with the python version as the user agent when the pip package is used within the worker?

Reply:
I don't want to expose the user agent publicly, as it should not be used publicly
In the worker, I would just set it via
```
client: ApiWorkflowClient
client.api_client.user_agent == f"Lightly/{__version__}/python ({platform.platform()}) ({worker_version})"
```

Reply:
Okey, thats a workable solution if its possible to import `__version__ ` from `lightly` 👍 

Maybe make the string
`f"Lightly/{__version__}/python (platform; {platform.platform()}) LightlyWorker/{worker_version}"`

Maybe you can also send some other information with the platform like the python version used? Wdyt? What could be valuable to know for us?

Reply:
Good idea to include the python version, then we can decide more easily in the future if we can deprecate support for an older version. I changed it to:
f"Lightly/{__version__}/python/{platform.python_version()} (platform; {platform.platform()})"

Reply:
> Okey, thats a workable solution if its possible to import __version__  from lightly

Yes, you can import it. I like your proposal with `LightlyWorker/{worker_version}`, but let's discuss it in the lightly-core PR.